tests/video_mediator_test.py

```diff
@@ -0,0 +1,9 @@
+import pytest
+from modules.videoMediator.videoMediator import VideoMediator
+
+
+# Integration test to check mediator behaviour
+def testVideoMediator():
```

Comment:
Don't think there's really much you can do for this module beyond integration testing